app/src/main/report/components/ReportingApp.tsx

```diff
@@ -1,36 +1,42 @@
 import * as React from "react";
-import { notificationStore } from "../../shared/stores/NotificationStore";
-import { connectToStores } from "../../shared/alt";
+import { connect } from 'react-redux';
+
 import { ErrorLog } from "../../shared/components/ErrorLog/ErrorLog";
 import { ReportingRouter } from "./ReportingRouter";
-import { reportingAuthStore } from "../stores/ReportingAuthStore";
 import { NotificationArea } from "../../shared/components/NotificationArea/NotificationArea";
 
+import { notificationStore } from "../../shared/stores/NotificationStore";
+import { connectToStores } from "../../shared/alt";
+
 export interface ReportingAppProps {
     errors: string[];
     infos: string[];
-    loggedIn: boolean;
 }
 
-export class ReportingAppComponent extends React.Component<ReportingAppProps, undefined> {
+export class ReportingAppComponent extends React.Component<any, undefined> {
     static getStores() {
-        return [ notificationStore, reportingAuthStore  ];
+        return [ notificationStore/*, reportingAuthStore */ ];
     }
     static getPropsFromStores(): ReportingAppProps {
         return {
             errors: notificationStore.getState().errors,
             infos: notificationStore.getState().infos,
-            loggedIn: reportingAuthStore.getState().loggedIn
         }
     }
 
-    render() {
+    render() :JSX.Element {
         return <div>
-            <ReportingRouter loggedIn={ this.props.loggedIn } />
+            <ReportingRouter loggedIn={ this.props.auth.loggedIn } />
             <NotificationArea notifications={ this.props.infos } />
             <ErrorLog errors={ this.props.errors } />
         </div>;
     }
 }
 
-export const ReportingApp = connectToStores(ReportingAppComponent);
\ No newline at end of file
+export const ReportingAppAltWrapped = connectToStores(ReportingAppComponent);
+
+const mapStateToProps = (state: any) => ({
+    auth: state.auth,
```

Comment:
So I think this can become 
``` const mapStateToProps = (state: GlobalState): Partial<ReportingAppProps> => ({
    loggedIn: state.auth.loggedIn
}); 
```